tests/test_pool.py

```diff
@@ -28,60 +25,49 @@ def __init__(self, *args, **kwargs):
         self.kwargs = kwargs
 
 
-class TestBackendPool(object):
-
-    def test_get_backends_should_return_all_backends_classes(self):
+class TestBackendPool:
+    @pytest.fixture
+    def configured_pool_with_fake_backend_with_constructor(self):
         configure(pools={
             BackendPool.backend_type: (
-                u'{module}.{cls}'.format(
-                    module=__name__,
-                    cls=FakeXYZBackend.__name__
-                ),
-                u'{module}.{cls}'.format(
+                '{module}.{cls}'.format(
                     module=__name__,
-                    cls=FakeABCBackend.__name__
+                    cls=FakeBackendWithConstructor.__name__
                 ),
             )
         })
 
-        backends = BackendPool.all_classes()
-
-        assert issubclass(backends[0], FakeXYZBackend)
-        assert issubclass(backends[1], FakeABCBackend)
-
-    def test_get_backend_should_return_a_backend_class(self):
+    @pytest.fixture
+    def configured_pools(self):
         configure(pools={
             BackendPool.backend_type: (
-                u'{module}.{cls}'.format(
+                '{module}.{cls}'.format(
                     module=__name__,
                     cls=FakeXYZBackend.__name__
                 ),
-                u'{module}.{cls}'.format(
+                '{module}.{cls}'.format(
                     module=__name__,
                     cls=FakeABCBackend.__name__
                 ),
             )
         })
 
+    def test_get_backends_should_return_all_backends_classes(
+        self,
+        configured_pools
+    ):
+        backends = BackendPool.all_classes()
+
+        assert issubclass(backends[0], FakeXYZBackend)
+        assert issubclass(backends[1], FakeABCBackend)
+
+    def test_get_backend_should_return_a_backend_class(self, configured_pools):
         backend = BackendPool.get_class('fake_abc')
         assert issubclass(backend, FakeABCBackend)
 
     def test_iterator_should_return_an_interator_with_all_backends_instances(
-        self
+        self, configured_pools
```

Comment:
```suggestion
        self,
        configured_pools
```

Reply:
I don't follow, maybe we can use [black](https://github.com/psf/black) for ramos and according to black the @jairhenrique ways is correct

Reply:
I don't know black framework yet. I just follow another arguments pattern on project